fabric-command-api-v2/src/client/java/net/fabricmc/fabric/impl/command/client/ClientCommandInternals.java

```diff
@@ -53,6 +54,7 @@ public final class ClientCommandInternals {
 	private static final String API_COMMAND_NAME = "fabric-command-api-v2:client";
 	private static final String SHORT_API_COMMAND_NAME = "fcc";
 	private static @Nullable CommandDispatcher<FabricClientCommandSource> activeDispatcher;
+	private static @Nullable CommandTreeS2CPacket lastReceivedCommandsPacket = null;
```

Comment:
Not keen on storing this staticlaly, as it may outlive the current connection. I think it would be best stored none statically in ClientPlayNetworkHandler.